src/xpk/parser/batch.py

```diff
@@ -39,3 +41,137 @@ def set_batch_parser(batch_parser):
   add_shared_arguments(batch_optional_arguments)
 
   batch_parser.set_defaults(func=batch)
+
+  batch_optional_arguments.add_argument(
+      '--ignore-unknown-flags',
+      type=bool,
+      action=argparse.BooleanOptionalAction,
+      default=False,
+      help='Ignore all the unsupported flags in the bash script.',
+  )
+  batch_optional_arguments.add_argument(
+      '-a',
+      '--array',
+      type=str,
+      default=None,
+      help=(
+          'Submit a job array, multiple jobs to be executed with identical'
+          ' parameters. The indexes specification identifies what array index'
+          ' values should be used. For example, "--array=0-15" or'
+          ' "--array=0,6,16-32". Multiple values may be specified using a comma'
+          ' separated list and/or a range of values with a "-" separator. For'
+          ' example "--array=0-15%%4" will limit the number of simultaneously'
+          ' running tasks from this job array to 4. The minimum index value is'
+          ' 0. The maximum index value is 2147483647.'
+      ),
+  )
+  batch_optional_arguments.add_argument(
+      '-c',
+      '--cpus-per-task',
+      type=str,
+      default=None,
+      help='How much cpus a container inside a pod requires.',
+  )
+  batch_optional_arguments.add_argument(
+      '--gpus-per-task',
+      type=str,
+      default=None,
+      help='How much gpus a container inside a pod requires.',
+  )
+  batch_optional_arguments.add_argument(
+      '--mem',
+      type=str,
+      default=None,
+      help='How much memory a pod requires.',
+  )
+  batch_optional_arguments.add_argument(
+      '--mem-per-task',
+      type=str,
+      default=None,
+      help='How much memory a container requires.',
+  )
+  batch_optional_arguments.add_argument(
+      '--mem-per-cpu',
+      type=str,
+      default=None,
+      help=(
+          'How much memory a container requires, it multiplies the number '
+          'of requested cpus per task by mem-per-cpu.'
+      ),
+  )
+  batch_optional_arguments.add_argument(
+      '--mem-per-gpu',
+      type=str,
+      default=None,
+      help=(
+          'How much memory a container requires, it multiplies the number '
+          'of requested gpus per task by mem-per-gpu.'
+      ),
+  )
+  batch_optional_arguments.add_argument(
+      '-N',
```

Comment:
-N for n. of nodes and -n for n of tasks is bad idea for me

Reply:
Slurm uses these flags https://slurm.schedmd.com/sbatch.html#OPT_nodes.